.travis.yml

```diff
@@ -24,7 +24,7 @@ cache:
   - directories: data
 install:
   - pip install docutils sphinx
-  - pip install -e .[test,louvain,leiden,magic,scvi]
+  - pip install -e .[test,louvain,leiden,magic,scvi,skmisc]
```

Comment:
The package on pypi is called `scikit-misc`.

Reply:
yes this is already addressed in setup.py, this is just a shortname